.gitignore

```diff
@@ -3,7 +3,6 @@
 # testing
 /coverage
 cypress/videos/
-cypress/fixtures/
```

Comment:
@john-m-liu fixtures is where mock data is stored and should be pushed to source control